client/src/portal/api/legacy_keyring.rs

```diff
@@ -0,0 +1,271 @@
+//! Legacy GNOME Keyring file format low level API.
+
+#[cfg(feature = "async-std")]
+use std::io;
+use std::{
+    collections::HashMap,
+    io::{Cursor, Read},
+};
+
+use endi::{Endian, ReadBytes};
+#[cfg(feature = "tokio")]
```

Comment:
This seems to be used only for errors, so let us just use std::io for that

Reply:
Right, switched to using std::io.